### PR TITLE
prep for 1.12.0

### PR DESCRIPTION
**Pull request by alanag13**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 The intended audience of this file is for py42 consumers -- as such, changes that don't affect
 how a consumer would use the library (e.g. adding unit tests, updating documentation, etc) are not captured here.
 
-## unreleased
+## 1.12.0 - 2021-02-25
 
 ### Added
```
Comment (unparalleled-js) [inline CHANGELOG.md:14]: note should be plural

Comment (unparalleled-js) [reply CHANGELOG.md:14]: Or have `a` first: 

``sdk.alerts.update_note()` to add or update _a_ note regarding an alert.`

Comment (unparalleled-js) [reply CHANGELOG.md:14]: to create _an_ `SDKClient`.

Comment (unparalleled-js) [reply CHANGELOG.md:14]: `mechanism` is a funny word lol

Comment (alanag13) [reply CHANGELOG.md:14]: 🙃

**Review by unparalleled-js — APPROVED** [review]

**Pull request merged**